README.md

```diff
@@ -109,8 +110,88 @@ MIT/Expat
+INFO:    zenodo:Updating .tributors cache from .zenodo.json
+WARNING:tributors:zenodo does not support updating from names.
+```
+
+In the case that there is more than one orcid found for a user, you will be given a list
+to check. Others will be updated in the file. You can then curate the file as you see fit.
+We next want to add the .allcontributors file:
+
+```bash
+$ tributors init allcontrib
+INFO:allcontrib:Generating .all-contributorsrc for datalad/datalad
+$ tributors update allcontrib
+INFO:allcontrib:Updating .all-contributorsrc
+INFO:allcontrib:Updating .tributors cache from .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor glalteva in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor adswa in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor chrhaeusler in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor soichih in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor mvdoc in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor mih in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor yarikoptic in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor loj in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor feilong in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor jhpoelen in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor andycon in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor nicholsn in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor adelavega in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor kskyten in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor TheChymera in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor effigies in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor jgors in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor debanjum in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor nellh in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor emdupre in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor aqw in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor vsoch in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor kyleam in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor driusan in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor overlake333 in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor akeshavan in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor jwodder in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor bpoldrack in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor yetanothertestuser in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor bhanuprasad14 in .all-contributorsrc
+WARNING:tributors:allcontrib does not support updating from orcids.
+WARNING:tributors:allcontrib does not support updating from email.
+```
```

Comment:
and how README.md would get updated for all-contributors?  should we enable all-contributors bot and ask it to do that or wait for con/tributors action to submit a PR with that?

Reply:
This is done with the GitHub workflow. For the individual categories (e.g., doc) the user would need to care and update the .all-contributorsrc with a PR after it's updated with their login.

Reply:
I'll add instructions for doing this.